lib/internal/inspector/_inspect.js

```diff
@@ -87,8 +87,8 @@ async function portIsFree(host, port, timeout = 9999) {
 
   setTimeout(timeout).then(() => ac.abort());
 
-  // eslint-disable-next-line no-unused-vars
-  for await (const _ of setInterval(retryDelay)) {
+  const asyncIterator = setInterval(retryDelay);
+  while (await asyncIterator.next()) {
```

Comment:
`const` and `next()` together??

Reply:
> `const` and `next()` together??

Yep, those work perfectly fine together.

```suggestion
  const asyncIterator = setInterval(retryDelay);
  while (!(await asyncIterator.next()).done) {
```

Reply:
@aduh95 I'm not opposed to your suggested change, but the reason I didn't do it that way is because writing it that way implies that `done` will evaluate to `true` at some point but that never actually happens, so I figured it was both more concise and more clear to write it the way I wrote it.

If you feel strongly that it's better the way you wrote it, I'm happy to go with it. But if you're not feeling strongly, I'll just leave things the way they are.

Reply:
Hum let's make it obvious we want an infinite loop then:
```suggestion
  const asyncIterator = setInterval(retryDelay);
  while (true) {
    await asyncIterator.next();
```